src/objects/Program_Enrollment__c.object

```diff
@@ -56,7 +56,7 @@
         <label>HEDA Program Enrollment Compact Layout</label>
     </compactLayouts>
     <deploymentStatus>Deployed</deploymentStatus>
-    <description>Contains details about the relationship a student Contact has with a Department or School (the degree-granting Account type for the University).</description>
```

Comment:
@juliesnow  Please note, I've updated the description of the Program Enrollment object to use the term "educational institution" instead of university.